platforms/core-configuration/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/plugins/precompiled/PrecompiledScriptPluginTemplatesTest.kt

```diff
@@ -65,6 +65,7 @@ import java.io.File
 @LeaksFileHandles("Kotlin Compiler Daemon working directory")
 class PrecompiledScriptPluginTemplatesTest : AbstractPrecompiledScriptPluginTest() {
 
+    @Suppress("DEPRECATION")
```

Comment:
can you just fix this to not use the deprecated API?

Reply:
For tests yes indeed